app/commands/git/sync_pull_requests.rb

```diff
@@ -0,0 +1,163 @@
+module Git
+  class SyncPullRequests
+    include Mandate
+
+    def call
+      repos.each do |repo|
+        @repo = repo
+        begin
+          sync!
+        rescue StandardError => e
+          Rails.logger.error "Error syncing pull requests for #{repo}: #{e}"
+        end
+      end
+    end
+
+    private
+    attr_reader :repo, :pull_requests, :cursor
+
+    def repos
+      page = 1
+      fetched = []
+
+      loop do
+        response = octokit_client.search_repositories("org:exercism is:public", page: page, per_page: 100)
+        fetched += response[:items].map { |item| item[:full_name] }
+        page += 1
+
+        break fetched if fetched.size >= response[:total_count]
+      end
+    end
+
+    def sync!
+      fetch!
+      create!
+    end
+
+    def create!
+      pull_requests.each do |pr|
+        ::GithubPullRequest.create_or_find_by!(github_id: pr[:pull_request][:id]) do |p|
+          p.github_username = pr[:pull_request][:user][:login]
+          p.github_repo = pr[:repository][:full_name]
+          p.github_event = pr
+        end
+      end
```

Comment:
I don't think we need `create_or_find_by` here as we're not using the resulting data. You can just do `create!` and catch the duplicate record exception. That should make this twice as quick as you don't need the extra finding sql and all the hundreds of objects Ruby uses to build a AR model.

Reply:
Makes sense!